### DISCUSSION TITLE
Dagga

### DISCUSSION THREAD

**Discussion by Gorgaki** [Thomasfds/intersect-cms-unleashed#22] [Add project]
Project Name : Dagga
Project Logo : https://www.ascensiongamedev.com/resources/filehost/21cd7869db68be847ed749ed5a840e7f.png
Project description : Dagga is an online rpg. The game is located on a continent where once lived a flourishing civilization. This place is full of ruins and dungeons to explore and loot! But beware of the creatures that live today there, they will not let you do so easily.
Project website url : https://site.daggaonline.fr/fr/

**Comment by Thomasfds** [comment]
Project add thank and sorry for late answer.
